conf/zeppelin-site.xml.template

```diff
@@ -396,4 +396,18 @@
 </property>
 -->
 
+<!--
+<property>
+  <name>zeppelin.server.strict.transport</name>
+  <value>max-age=631138519</value>
+  <description>The HTTP Strict-Transport-Security response header is a security feature that lets a web site tell browsers that it should only be communicated with using HTTPS, instead of using HTTP. Enable this when Zeppelin is running on HTTPS.Set value is in Seconds equivalent to 20 years.</description>
```

Comment:
should this say "Set value is in Seconds equivalent to 20 years"
-> "Value is in Seconds, the default value is equivalent to 20 years"